packages/core/src/tracing/instrumentation/frameworks/express.js

```diff
@@ -30,18 +30,26 @@ exports.init = function init() {
 function instrument(express) {
   if (express.Router && express.Router.handle && express.Router.use) {
     // express 4
+    // capture the uncaught error
     shimmer.wrap(express.Router, 'handle', shimExpress4Handle);
     shimmer.wrap(express.Router, 'use', shimExpress4Use);
   }
 
   if (express.Route && express.Route.prototype) {
-    // express 4
+    // express v4, v5 beta
+    // capture the path template
     methods.concat('all').forEach(method => {
       if (typeof express.Route.prototype[method] === 'function') {
         shimmer.wrap(express.Route.prototype, method, shimHandlerRegistration);
       }
     });
   }
+  if (express.Router && express.Router.prototype) {
```

Comment:
issue: 

> if (express.Router && express.Router.prototype) {

This is `true` for express v4. The logic should not get triggered when express is v4.